types/hash.go

```diff
@@ -75,9 +75,6 @@ func (h Hash) Hex() string {
 
 // BigInt returns the *big.Int representation of the *Hash
 func (h *Hash) BigInt() *big.Int {
-	if new(big.Int).SetBytes(ReverseByteOrder(h[:])) == nil {
```

Comment:
This would never happen.

Reply:
Yes. I think it is right. And I also prefer `big.NewInt(0)` to `new(big.Int)`

Reply:
changed to `big.NewInt(0)`